src/main/java/io/github/darealturtywurty/turtychemistry/block/FoundryBlock.java

```diff
@@ -0,0 +1,50 @@
+package io.github.darealturtywurty.turtychemistry.block;
+
+import io.github.darealturtywurty.turtychemistry.TurtyChemistry;
+import io.github.darealturtywurty.turtychemistry.block.entity.FoundryBlockEntity;
+import io.github.darealturtywurty.turtychemistry.init.BlockEntityInit;
+import io.github.darealturtywurty.turtychemistry.menu.FoundryMenu;
+import net.minecraft.core.BlockPos;
+import net.minecraft.network.chat.Component;
+import net.minecraft.server.level.ServerPlayer;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.SimpleMenuProvider;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.BaseEntityBlock;
+import net.minecraft.world.level.block.entity.BlockEntity;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.phys.BlockHitResult;
+import net.minecraftforge.network.NetworkHooks;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+public final class FoundryBlock extends BaseEntityBlock {
```

Comment:
```suggestion
public final class FoundryBlock extends Block implements EntityBlock {
```

Reply:
isn't extending block and implementing entity block just the same as just extending BaseEntityBlock?

Reply:
No

Reply:
Extending that class also changes the render shape whic can cause unwanted consequences